app/models/permission.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: permissions
+#
+#  id          :bigint           not null, primary key
+#  description :string
+#  permission  :string           not null
+#  object_type :string           not null
+#  object_pk   :integer          not null
+#  user_id     :bigint
+#  group_id    :bigint
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# The permissions system provides a way to assign specific permissions
+# to users and groups of users for specific objects. The object_type
+# is intended to be either a Resource or Service and the object_pk
```

Comment:
Thanks for writing this comment and for adding the schema in comments! I miss being able to look see the schema at a glance without having to open up schema.rb!

Could we document specifically what `object_type` should look like? Is it literally the strings `"Resource"` and `"Service"`, spelled with a capital initial letter and in singular form? With Rails its a little hard to tell what the canonical name for a table is, since the models are generally spelled in singular form, while the tables are spelled in plural form (and if you use a word with an irregular plural, you end up with weird things like "feedbacks", which isn't technically a word). If they're supposed to be `"Resource"` and `"Service"`, maybe you could put double quotes around them in the comment to more strongly suggest that those are literally what the string value should be?